### DISCUSSION TITLE
How to specify main folder in Javet?

### DISCUSSION THREAD

**Discussion by Arothrone** [caoccao/Javet#218] [Q&A]
I made a chat translation plugin for minecraft, which is based on a combination of java and javascript. But the server at startup requires administrator rights for the plugin to work correctly. I realized that maybe the problem is that the plugin reads "node_modules" in "...\server1.19", not in "...\server1.19\plugins\Translatechat". How do I set a different folder as the main folder in Javet? (Google translate.)

I tried reading the Javet docs, but I'm not a very good programmer to understand anything there.

I expected that plugin will work good without administator rights.

But it still requires admin rights.

**Comment by caoccao** [comment]
It is the current directory of JVM. So, if you start the JVM from a non-privileged directory sitting with the node_modules side by side, you will likely get it work.
